tests/unit/test_hyper_transformer.py

```diff
@@ -2202,6 +2197,33 @@ def test_update_sdtypes_different_sdtype_transformer_provided(self, mock_warning
         assert instance.field_transformers == {'a': transformer}
         instance._user_message.assert_called_once_with(user_message, 'Info')
 
+    def test__validate_transformers(self):
```

Comment:
Can we add a line to `test_update_transformers` that makes sures the proper values are passed to `_validate_transformers` and that it is called?